src/components/Images.vue

```diff
@@ -25,16 +25,17 @@
 		:mime="mime"
 		:src="src"
 		:fileid="fileid"
+		@updated="onUpdate"
 		@close="onClose" />
 
-	<img v-else
+	<img v-else-if="data !== null"
```

Comment:
Needed as we are using a concatenation for the `src` prop which seem to interfere with `AsyncComputed` logic.
The issue was that a request with `null` was sent instead of waiting for `data` to be initialized.